### PR TITLE
Release notes for v1.6.3

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

This targets the v1.6 branch for now, we can cherry pick into master after release.

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @baude @TomSweeneyRedHat @vrothberg @haircommander PTAL and merge

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # Release Notes
 
+## 1.6.3
+### Features
+- Handling of the `libpod.conf` configuration file has seen major changes. Most significantly, rootless users will no longer automatically receive a complete configuration file when they first use Podman, and will instead only receive differences from the global configuration.
+- Initial support for the CNI DNS plugin, which allows containers to resolve the IPs of other containers via DNS name, has been added
+- Podman now supports anonymous named volumes, created by specifying only a destination to the `-v` flag to `podman create` and `podman run`
+- Named volumes now support `uid` and `gid` options in `--opt o=...` to set UID and GID of the created volume
```
Comment (rhatdan) [inline RELEASE_NOTES.md]: @nalind Should mention something about pull by digest?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed a bug where Podman would not error if the `systemd` CGroup manager was specified, but systemd could not be contacted over DBus
+- Fixed a bug where image volumes were mounted `noexec` ([#4318](https://github.com/containers/libpod/issues/4318))
+- Fixed a bug where the `podman stats` command required the name of a container to be given, instead of showing all containers when no container is specified ([#4274](https://github.com/containers/libpod/issues/4274))
+- Fixed a bug where the `podman volume inspect` command would not show the options that named volumes were created with
+- Fixed a bug where custom storage configuration was not written to `storage.conf` at time of first creation for rootless Podman ([#2659](https://github.com/containers/libpod/issues/2659))
+- Fixed a bug where remote Podman did not support shell redirection of container output
+
+### Misc
+- Updated vendored containers/image library to v5.0
+- Initial support for images using manifest lists has been added, though commands for directly interacting with manifests are still missing
```
Comment (rhatdan) [inline RELEASE_NOTES.md]: I think you can now pull directly by manifest, we just don't allow you to play with them yet.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
 # Release Notes
 
+## 1.6.3
+### Features
+- Handling of the `libpod.conf` configuration file has seen major changes. Most significantly, rootless users will no longer automatically receive a complete configuration file when they first use Podman, and will instead only receive differences from the global configuration.
+- Initial support for the CNI DNS plugin, which allows containers to resolve the IPs of other containers via DNS name, has been added
+- Podman now supports anonymous named volumes, created by specifying only a destination to the `-v` flag to `podman create` and `podman run`
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: add 'commands' to the end here.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed a bug where incorrect Seccomp profiles were used in containers generated by `podman play kube`
+- Fixed a bug where processes started by `podman exec` would have the wrong SELinux label in some circumstances ([#4361](https://github.com/containers/libpod/issues/4361))
+- Fixed a bug where error messages from `slirp4netns` would be lost
+- Fixed a bug where `podman run --network=$NAME` would not throw an error in rootless Podman, where CNI networks are not supported
+- Fixed a bug where `podman network create` would throw confusing errors when trying to create a volume with a name that already exists
+- Fixed a bug where Podman would not error if the `systemd` CGroup manager was specified, but systemd could not be contacted over DBus
+- Fixed a bug where image volumes were mounted `noexec` ([#4318](https://github.com/containers/libpod/issues/4318))
+- Fixed a bug where the `podman stats` command required the name of a container to be given, instead of showing all containers when no container is specified ([#4274](https://github.com/containers/libpod/issues/4274))
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: s/was specified/is specified/

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed a bug where the `podman volume inspect` command would not show the options that named volumes were created with
+- Fixed a bug where custom storage configuration was not written to `storage.conf` at time of first creation for rootless Podman ([#2659](https://github.com/containers/libpod/issues/2659))
+- Fixed a bug where remote Podman did not support shell redirection of container output
+
+### Misc
+- Updated vendored containers/image library to v5.0
+- Initial support for images using manifest lists has been added, though commands for directly interacting with manifests are still missing
+- Support for pushing to and pulling from OSTree has been removed due to deprecation in the containers/image library
+- Rootless Podman no longer enables linger on systems with systemd as init by default. As such, containers will now be killed when the user who ran them logs out, unless linger is explicitly enabled
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: can you talk about how linger could be reenabled?

**Comment by mheon** [comment]
Updated

**Pull request merged**